pkg/controller/elasticsearch/driver/upgrade_test.go

```diff
@@ -45,8 +45,14 @@ func Test_podsToUpgrade(t *testing.T) {
 			name: "all pods need to be upgraded",
 			args: args{
 				statefulSets: sset.StatefulSetList{
-					sset.TestSset{Name: "masters", Replicas: 2, Master: true, Status: appsv1.StatefulSetStatus{CurrentRevision: "rev-a", UpdateRevision: "rev-b"}}.Build(),
-					sset.TestSset{Name: "nodes", Replicas: 3, Master: true, Status: appsv1.StatefulSetStatus{CurrentRevision: "rev-a", UpdateRevision: "rev-b"}}.Build(),
+					sset.TestSset{
+						Name: "masters", Replicas: 2, Master: true,
+						Status: appsv1.StatefulSetStatus{CurrentRevision: "rev-a", UpdateRevision: "rev-b", UpdatedReplicas: 0, Replicas: 2},
```

Comment:
Should we remove any misleading ref to `CurrentRevision` and `UpdateRevision` from the tests since they don't matter at all anymore?

Reply:
Well they still do matter because this test looks at `ToUpdate` + the actual revisions on the individual pods.